project-generator/src/test/resources/io/syndesis/project/converter/test-pull-push-pom.xml

```diff
@@ -16,6 +16,7 @@
     <camel.version>@camel.version@</camel.version>
     <funktion.version>@funktion.version@</funktion.version>
     <atlasmap.version>@atlasmap.version@</atlasmap.version>
+    <syndesis-connectors.version>@syndesis-connectors.version@</syndesis-connectors.version>
```

Comment:
afais the ${syndesis-connectors.version} parameter below is replaced when the resources are copied over with the value from the pom.xml so that when the tests runs its already resolved and no property statement is included but the literal version numbers.

But I could be wrong here, so would be worth a try.